content-accelerator/latest/using/user-guide.md

```diff
@@ -1154,6 +1155,65 @@ you to search for all possible documents in the repository.
+
+To enable advanced capabilities in attribute search, complete the following steps.
+
+1. Navigate to the Attribute Search config in the Alfresco Content Accelerator Admin and set the value of the Show Advanced Search parameter to Enabled. By default, the value is set to Disabled.
+
+    ![Advanced Search configuration]({% link content-accelerator/images/aca-show-advanced-search-config.png %})
+
+2. After you enable Advanced Search, you can enable the following configuration settings:
+  * Enable Any/All Search
+  * Enable Like/Exact/Not Search
+
+    ![Advanced Search additional configuration]({% link content-accelerator/images/aca-show-advanced-search-options-config.png %})
+
+When the Enable Any/All Search setting is enabled, you can configure either of the following settings.
+
+| Settings | Search operator |
+| -------- | --------------- |
+| ANY of these values | If you select this option, search is performed based on the OR operator |
+| ALL of these values |	If you select this option, search is performed based on the AND operator |
+
+
+When you enable the **Like/Exact/Not Search** setting, an additional control is added to the Attribute Search form. The control includes an advanced operator selection drop-down. This drop-down is only available to properties with control type `TextBox`, `AutoComplete`, and `DropDown`.
+
+The list of operators included in the drop-down is based on the associated property data type. The following table lists the data types and the operators associated with it.
+
+| Data type | Operators |
+| --------- | --------- |
+| String | LOGIC_LIKE, OPERATOR_EQUALS, OPERATOR_NOT_EQUALS |
+
+| integer or double | OPERATOR_LESS_THAN, OPERATOR_EQUALS, OPERATOR_GREATER_THAN, OPERATOR_NOT_EQUALS |
+
+The operators listed above are described below:
+
+* LOGIC_LIKE - Performs a wildcard (%) search 
+* OPERATOR_EQUALS - Performs a exact (=) value search
+* OPERATOR_NOT_EQUALS - Performs a not exact (-) value search
+* OPERATOR_LESS_THAN - The search returns results less than (<) the provided search term.
+* OPERATOR_GREATER_THAN - The search will return results greater than (>) the provided search term.
+
+If the Like/Exact/Not Search setting is disabled, the default operation depends on the object type. The following table lists the default operation based on the object types.
+
+
+| Object | Default operation |
+| ------ | ----------------- |
+| **AutoComplete** and **DropDown** | Exact search with case sensitivity |
+| **RadioButton** and **Cascading** | Exact search |
+| **TextBox** | The search depends on values set using “Make Search Exact” and “Case Sensitive Search” and if these values are not set, a wildcard search is performed. |
+
+**Notes:**
+
```

Comment:
please mention it is available as on ACA 3.6.x .

Reply:
Advanced Search is deprecated in Content Accelerator 3.6. Though it is still available, it is expected to be removed in future releases and replaced with Advanced Search capabilities added to Attribute Search, described above.